pkg/testutils/docker/docker_test: TestSingleNodeDocker failed

pkg/testutils/docker/docker_test.TestSingleNodeDocker [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7564132?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7564132?buildTab=artifacts#/) on master @ [ea7c52e5a930be203318db6e338edd5dde7537bd](https://github.com/cockroachdb/cockroach/commits/ea7c52e5a930be203318db6e338edd5dde7537bd):


```
=== RUN   TestSingleNodeDocker
    single_node_docker_test.go:187: cannot list all containers on docker image cockroachdb/cockroach-ci:latest: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
--- FAIL: TestSingleNodeDocker (0.00s)
=== RUN   TestSingleNodeDocker/single-node-insecure-mem-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-insecure-mem-mode (0.00s)
=== RUN   TestSingleNodeDocker/single-node-insecure-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-insecure-mode (0.00s)
=== RUN   TestSingleNodeDocker/single-node-secure-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-secure-mode (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSingleNodeDocker.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-21551

pkg/testutils/docker/docker_test.TestSingleNodeDocker [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7564137?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7564137?buildTab=artifacts#/) on master @ [ea7c52e5a930be203318db6e338edd5dde7537bd](https://github.com/cockroachdb/cockroach/commits/ea7c52e5a930be203318db6e338edd5dde7537bd):


```
=== RUN   TestSingleNodeDocker
    single_node_docker_test.go:187: cannot list all containers on docker image cockroachdb/cockroach-ci:latest: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
--- FAIL: TestSingleNodeDocker (0.00s)
=== RUN   TestSingleNodeDocker/single-node-insecure-mem-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-insecure-mem-mode (0.00s)
=== RUN   TestSingleNodeDocker/single-node-insecure-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-insecure-mode (0.00s)
=== RUN   TestSingleNodeDocker/single-node-secure-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-secure-mode (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSingleNodeDocker.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/testutils/docker/docker_test.TestSingleNodeDocker [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7564140?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7564140?buildTab=artifacts#/) on master @ [ea7c52e5a930be203318db6e338edd5dde7537bd](https://github.com/cockroachdb/cockroach/commits/ea7c52e5a930be203318db6e338edd5dde7537bd):


```
=== RUN   TestSingleNodeDocker
    single_node_docker_test.go:187: cannot list all containers on docker image cockroachdb/cockroach-ci:latest: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
--- FAIL: TestSingleNodeDocker (0.01s)
=== RUN   TestSingleNodeDocker/single-node-insecure-mem-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-insecure-mem-mode (0.00s)
=== RUN   TestSingleNodeDocker/single-node-insecure-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-insecure-mode (0.00s)
=== RUN   TestSingleNodeDocker/single-node-secure-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-secure-mode (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSingleNodeDocker.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/testutils/docker/docker_test.TestSingleNodeDocker [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7582667?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7582667?buildTab=artifacts#/) on master @ [a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412](https://github.com/cockroachdb/cockroach/commits/a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412):


```
=== RUN   TestSingleNodeDocker
    single_node_docker_test.go:187: cannot list all containers on docker image cockroachdb/cockroach-ci:latest: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
--- FAIL: TestSingleNodeDocker (0.00s)
=== RUN   TestSingleNodeDocker/single-node-insecure-mem-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-insecure-mem-mode (0.00s)
=== RUN   TestSingleNodeDocker/single-node-insecure-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-insecure-mode (0.00s)
=== RUN   TestSingleNodeDocker/single-node-secure-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-secure-mode (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSingleNodeDocker.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/testutils/docker/docker_test.TestSingleNodeDocker [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7582675?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7582675?buildTab=artifacts#/) on master @ [a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412](https://github.com/cockroachdb/cockroach/commits/a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412):


```
=== RUN   TestSingleNodeDocker
    single_node_docker_test.go:187: cannot list all containers on docker image cockroachdb/cockroach-ci:latest: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
--- FAIL: TestSingleNodeDocker (0.01s)
=== RUN   TestSingleNodeDocker/single-node-insecure-mem-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-insecure-mem-mode (0.00s)
=== RUN   TestSingleNodeDocker/single-node-insecure-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-insecure-mode (0.00s)
=== RUN   TestSingleNodeDocker/single-node-secure-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-secure-mode (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSingleNodeDocker.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/testutils/docker/docker_test.TestSingleNodeDocker [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7582672?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7582672?buildTab=artifacts#/) on master @ [a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412](https://github.com/cockroachdb/cockroach/commits/a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412):


```
=== RUN   TestSingleNodeDocker
    single_node_docker_test.go:187: cannot list all containers on docker image cockroachdb/cockroach-ci:latest: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
--- FAIL: TestSingleNodeDocker (0.00s)
=== RUN   TestSingleNodeDocker/single-node-insecure-mem-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-insecure-mem-mode (0.00s)
=== RUN   TestSingleNodeDocker/single-node-insecure-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-insecure-mode (0.00s)
=== RUN   TestSingleNodeDocker/single-node-secure-mode
    single_node_docker_test.go:207: cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
    --- FAIL: TestSingleNodeDocker/single-node-secure-mode (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSingleNodeDocker.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/build/teamcity/cockroach/nightlies/stress_impl.sh
+++ b/build/teamcity/cockroach/nightlies/stress_impl.sh
@@ -31,6 +31,11 @@ do
         echo "Skipping test $test as it is broken in bazel"
         continue
     fi
+    if [[ ! -z $(bazel query "attr(tags, \"integration\", $test)") ]]
+    then
+        echo "Skipping test $test as it is an integration test"
+        continue
+    fi
     exit_status=0
     $BAZEL_BIN/pkg/cmd/bazci/bazci_/bazci -- test --config=ci "$test" \
                                           --test_env=COCKROACH_NIGHTLY_STRESS=true \

